beacon-chain/rpc/validator/proposer.go

```diff
@@ -154,6 +155,9 @@ func (vs *Server) ProposeBlock(ctx context.Context, blk *ethpb.SignedBeaconBlock
 //  - Subtract that eth1block.number by ETH1_FOLLOW_DISTANCE.
 //  - This is the eth1block to use for the block proposal.
 func (vs *Server) eth1Data(ctx context.Context, slot uint64) (*ethpb.Eth1Data, error) {
+	ctx, cancel := context.WithTimeout(ctx, 2*time.Second)
```

Comment:
Can you move this to a global var? Rather than hard-coded into the function.

Reply:
Done